docker-compose.yml

```diff
@@ -3,23 +3,21 @@ services:
   haystack-api:
     # Pull Haystack's latest commit
     image: "deepset/haystack:cpu-main"
+    volumes:
+      - <PATH_TO_HAYSTACK>/haystack/rest_api/rest_api/pipeline:/home/user/rest_api/pipeline
```

Comment:
I think the compose file should work out of the box: why not mounting 
```
./rest_api/rest_api/pipeline:/home/user/rest_api/pipeline
```

Another thing we might make easier to understand is the mount target: instead of `/home/user/rest_api/pipeline` we could use something like `/opt/pipelines` - same effect, just we send the message pipeline files can be wherever in the filesystem and not necessarily in the `home` folder or anything `rest_api` related.

The result would be:
```
./rest_api/rest_api/pipeline:/opt/pipelines
```